### PR TITLE
[dashboard] allow to open desktop links on app domain

### PR DESCRIPTION
**Pull request by akosyakov**
## Description

It is supporting PR for https://github.com/gitpod-io/gitpod/pull/12082. It should be deployed by the webapp team before.

## Related Issue(s)

Fixes #

## How to test

Test https://github.com/gitpod-io/gitpod/pull/12082

## Release Notes

```release-note
NONE
```

## Werft options:

- [x] /werft with-preview

### CODE REVIEW

**Comment by laushinka** [comment]
@akosyakov I followed the steps [here](https://github.com/gitpod-io/gitpod/pull/12082#issue-1336823421), but when I opened another workspace, I was still prompted by the pop-up.

Screenshot 2022-08-15 at 13 15 43
.

**Comment by akosyakov** [comment]
@laushinka you should use prev env of another PR. This PR only bring necessary changes which should be deployed before by webapp team.

**Comment by laushinka** [comment]
> @laushinka you should use prev env of another PR

@akosyakov I guess [this PR](https://github.com/gitpod-io/gitpod/pull/12082) is already tested and reviewed. What behavior should I test on this one?

File: components/dashboard/src/start/StartWorkspace.tsx
Comment (laushinka) [inline components/dashboard/src/start/StartWorkspace.tsx:400]: Should be `openDesktopLink` instead of `Dekstop`

**Comment by akosyakov** [comment]
> @akosyakov I guess https://github.com/gitpod-io/gitpod/pull/12082 is already tested and reviewed. What behavior should I test on this one?

mostly code review, you can check for regressions but you did it already, ie it still works like on main

if all good I would generally approve and /hold on new line to let author address typos, nits and merge later 🙏🏻

**Comment by laushinka** [comment]
Approving with a hold to address comments later.

/hold

**Review by laushinka — APPROVED** [review]

**Pull request merged**
